src/task.rs

```diff
@@ -607,27 +611,32 @@ impl<'a> Runner<'a> {
                     }
                     None => target.to_string(),
                 };
+                dbg!(&self.nodes.len(), &self.nodes, num);
+                /* if (self.file_nodes.get(&file_target).is_none) {
+                    return ;
+                } */
                 match self.file_nodes.get(&file_target) {
```

Comment:
Note that file nodes needs to be uniquely keyed by unique file path strings. If you try logging `self.file_nodes`, it could be worth checking that the files are all absolute path keys in there.